script/websocket-server.rb

```diff
@@ -120,7 +120,11 @@ def after_fork(dir)
 @clients = {}
 Rails.configuration.interface = 'websocket'
 EventMachine.run do
-  EventMachine::WebSocket.start( host: @options[:b], port: @options[:p], secure: @options[:s], tls_options: tls_options ) do |ws|
+
+  if /(\[(?:[:0-9A-Fa-f]+)\])/.match?(@options[:b]) # check if there is an IPv6 Address
```

Comment:
Checking ipv6 adresses by own regex looks wrong to me.
We could use resolv here: http://ruby-doc.org/stdlib-2.4.1/libdoc/resolv/rdoc/Resolv/IPv6.html
Example: https://stackoverflow.com/questions/3634998/how-do-i-check-whether-a-value-in-a-string-is-an-ip-address/15157862#15157862